task/06-conditions-n-loops-tasks.js

```diff
@@ -166,7 +193,11 @@ function isInsideCircle(circle, point) {
  *   'entente' => null
  */
 function findFirstSingleChar(str) {
-    throw new Error('Not implemented');
+    for (let i = 0; i < str.length; i++)
+    if ((str.indexOf(str[i]) == i) && (str.indexOf(str[i], i + 1) == -1)) {
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
"0" == 0
and "0" === 0)"